### DISCUSSION TITLE
Format of returning data is change ?

### DISCUSSION THREAD

**Discussion by Khanhan2k2** [supabase/supabase#14475] [Questions]
Hi everyone, I have met a problem, I will describe my db first, I have a users table and shops table, user_id field of shops table reference to id field of users table, last week I coded a feature that need query shop from user, and I write this 
![image](https://github.com/supabase/supabase/assets/100439816/e6214551-e67c-4163-8afb-e6db12132504)
then shops return is an array
but today it return an object, so is the format of returned data is changed ?
Thankfully

**Comment by steve-chavez** [comment]
If the relationship between users and shops is a one-to-one then shops will be an object. More details at https://postgrest.org/en/stable/references/api/resource_embedding.html#one-to-one-relationships

This is not a recent change. It was done past year: https://supabase.com/blog/postgrest-v10#one-to-one-relationships

Perhaps you were on an old instance until now?
